packages/x-components/src/types/result-location.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Indicates where the result where the event comes from is located.
+ *
+ * @public
+ */
+export type ResultLocation =
+  | 'recommendations'
```

Comment:
I know we talked yesterday about using `recommendations` here because if the reccommendations come without origin they should be the ones in the `no_results` situation, but I would name them `no_query`. How are we going to distinguish them from the ones in the `predictive_layer`?

Reply:
Done, merged both now that it is tracked in the same way